bagit.py

```diff
@@ -44,10 +44,16 @@
 from datetime import date
 from os import listdir
 from os.path import abspath, isdir, isfile, join
+from pkg_resources import DistributionNotFound, get_distribution
 
-LOGGER = logging.getLogger(__name__)
+MODULE_NAME = 'bagit' if __name__ == '__main__' else __name__
```

Comment:
In case this isn't obvious, it's just to support what happens when someone runs `python bagit.py` directly since `__main__` is both very generic and will break the version parsing below.